src/strategies/lrc-l2-subgraph-balance-of/index.ts

```diff
@@ -0,0 +1,91 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { subgraphRequest } from '../../utils';
+
+export const author = 'shad-k';
+export const version = '0.1.1';
+
+function makeQuery(snapshot, addresses, tokenId) {
+  const query: any = {
+    accounts: {
+      __args: {
+        where: {
+          address_in: addresses
+        }
+      },
+      balances: {
+        __args: {
+          where: {
+            token_in: [`${tokenId}`]
+          }
+        },
+        balance: true,
+        token: {
+          decimals: true
+        }
+      },
+      address: true
+    }
+  };
+
+  if (snapshot !== 'latest') {
+    query.accounts.__args = {
+      ...query.accounts.__args,
+      block: {
+        number: snapshot
+      }
+    };
+  }
+
+  return query;
+}
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const LIMIT = 100;
```

Comment:
Hi, @shad-k Is there a chance to increase this limit? Should be fine in many cases but want to know if there is a possibility to make it at least 500

Reply:
Hi @ChaituVR , ran some tests to check if we can increase the limit to 500. Changed the limit to 500.